test/HealthChecks.Publisher.Seq.Tests/Functional/SeqPublisherTests.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Net;
+
+namespace HealthChecks.Publisher.Seq.Tests.Functional;
+
+public class seq_publisher_should
+{
+    [Fact]
+    public async Task handle_trailing_slash_in_endpoint()
+    {
+        var options = new SeqOptions
+        {
+            ApiKey = "test-key",
+            Endpoint = "http://localhost:5341/"
+        };
+
+        var expectedUri = new Uri("http://localhost:5341/api/events/raw?apiKey=test-key");
+
+        // Setup mocks
+        using var handler = new MockClientHandler();
+        HttpClient HttpClientFactory() => new(handler);
+
+        var testReport = new HealthReport(new Dictionary<string, HealthReportEntry>(), TimeSpan.Zero);
+
+        // Create publisher and publish
+        var publisher = new SeqPublisher(HttpClientFactory, options);
+        await publisher.PublishAsync(testReport, CancellationToken.None);
+
+        Assert.NotNull(handler.Request);

```

Comment:
Please use FluentAssertions extension methods instead of Assert.

Reply:
done